pkg/helper/bridges_test.go

```diff
@@ -78,7 +78,7 @@ var (
 
 var _ = Describe("Network desired state bridge parser", func() {
 	var (
-		obtainedBridges []string
+		obtainedBridges map[string][]string
```

Comment:
please rename the variable to reflect what does it keep, it is not only bridge now